### ISSUE TITLE
CacULE/v5.16/0001-cacULE-5.16-full.patch 404 error

### ISSUE THREAD

**Issue by iamtrazy** [Frogging-Family/linux-tkg#416]
https://raw.githubusercontent.com/CachyOS/cacule-cpu-scheduler/master/patches/CacULE/v5.16/0001-cacULE-5.16-full.patch
SSL_INIT
Loaded CA certificate '/etc/ssl/certs/ca-certificates.crt'
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.109.133, 185.199.108.133, 185.199.110.133, ...
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.109.133|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2022-01-31 09:51:24 ERROR 404: Not Found.

there is no patch named 0001-cacULE-5.16-full.patch in cachyOS repo https://github.com/CachyOS/cacule-cpu-scheduler/tree/master/patches/CacULE/v5.16

**Comment by iamtrazy** [comment]
is it okay if i use https://github.com/CachyOS/cacule-cpu-scheduler/blob/master/patches/CacULE/v5.16/0001-cacULE-5.16.patch , i think there is a issue with naming scheme

**Comment by iamtrazy** [comment]
https://github.com/CachyOS/cacule-cpu-scheduler/commit/c28a7acf306a9d7ebcb2adb124acce6fb585ce0d i think CachyOS renamed there patch in this commit https://github.com/CachyOS/cacule-cpu-scheduler/commit/c28a7acf306a9d7ebcb2adb124acce6fb585ce0d

**Comment by ptr1337** [comment]
> I think CachyOS renamed there patch in this commit [CachyOS/cacule-cpu-scheduler@c28a7ac](https://github.com/CachyOS/cacule-cpu-scheduler/commit/c28a7acf306a9d7ebcb2adb124acce6fb585ce0d)

Yes, cause his commit was the weird naming scheme, so i did change it. 
I also wrote TKG that i changed it. So its the same name scheming.

**Comment by Orangian** [comment]
Ah. In that case lines 553-569 in prepare can be restructured to no longer check for kernel versions above 5.15.

**Comment by ryaniskira** [comment]
@Orangian Line 568 needs to be changed as well.

**Comment by AdelKS** [comment]
Oh this issue can be closed ! Good job guys
